api/openapi-spec/openapi.yaml

```diff
@@ -306,23 +306,57 @@ components:
           type: string
         key:
           type: string
+        audiences:
+          type: string
         variationsMap:
           type: object
           additionalProperties:
             $ref: '#/components/schemas/Variation'
       required:
         - id
         - key
+    Attribute:
+      properties:
+        id:
+          type: string
+        key:
+          type: string
+    OptimizelyAudience:
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        conditions:
+          type: string
+    OptimizelyEvent:
+      properties:
+        id:
+          type: string
+        key:
+          type: string
+        experimentIds:
+          type: array
+          items:
+            type: string
     Feature:
       properties:
         id:
           type: string
         key:
           type: string
+        experimentRules:
+          type: array
+          items:
+            $ref: '#/components/schemas/Experiment'
```

Comment:
Should these be OptimizelyExperiments or does it follow the same as Experiment from config? Same for feature and attribute.